ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm

```diff
@@ -14,3 +14,9 @@
 	template_height = 6
 	stock = 3
 	room_type = "backstreets_medium_north"
+
+ /datum/map_template/random_room/backstreets/medium_north/workshop_mn
+    name = "Workshop - Medium North"
```

Comment:
Got an extra space in here, just make it one tab

Reply:
think I got it, please check